### ISSUE TITLE
Ratelimiter API: Return error for invalid fields in PUT/PATCH requests 

### ISSUE THREAD

**Issue by sandreim** [firecracker-microvm/firecracker#1437]
Currently this invalid PATCH request ( **bytes** instead of **bandwidth**) does not yield an error:
```bash
  sudo curl -X PATCH -i \
  --unix-socket /tmp/firecracker-andrei.socket \
  http://localhost/network-interfaces/eth0 \
  -H  'Accept: application/json'          \
  -H  'Content-Type: application/json'    \
  -d '{
    "iface_id": "eth0",
      "rx_rate_limiter": {
      "bytes": {
            "size": 62500,
            "refill_time": 1000
          }
      },
      "tx_rate_limiter": {
          "bytes": {
            "size": 62500,
            "refill_time": 1000
          }
      }
  }'
```
The log shows:
2019-11-27T13:40:54.952166781 [anonymous-instance] The Patch request on "/network-interfaces/eth0" with body "{\n    \"iface_id\": \"eth0\",\n      \"rx_rate_limiter\": {\n      \"bytes\":
 {\n            \"size\": 62500,\n            \"refill_time\": 1000\n          }\n      },\n      \"tx_rate_limiter\": {\n          \"bytes\": {\n            \"size\": 62500,\n
\"refill_time\": 1000\n          }\n      }\n  }" was executed successfully. Status code: 204 No Content.

**Comment by pyrito** [comment]
Not sure if someone is working on this but @mi-yu and I worked on a fix for this?

**Comment by sandreim** [comment]
Hi @pyrito, unfortunately this issue is already assgined and @andrei-cosmin is already working on it. As a tip for finding a good issue please look at those with the `Contribute: help wanted` and `Contribute: good first issue` labels. 

For example: https://github.com/firecracker-microvm/firecracker/issues/745

**Comment by serban300** [comment]
Fixed by #1441
